.opencode/commands/career-ops-pdf2.md

```
@@ -0,0 +1,10 @@
+---
+description: Batch PDF generation for multiple jobs in parallel
+---
+
+Generate ATS-optimized CV PDFs for multiple jobs in parallel using career-ops pdf2 mode.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)